client/src/Movies/MovieList.js

```diff
@@ -33,22 +34,22 @@ export default class MovieList extends Component {
 
 function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
-  return (
-    <div className="movie-card">
-      <h2>{title}</h2>
-      <div className="movie-director">
-        Director: <em>{director}</em>
-      </div>
-      <div className="movie-metascore">
-        Metascore: <strong>{metascore}</strong>
-      </div>
-      <h3>Actors</h3>
+    return (
+	<Link to={`/movies/${movie.id}`} className="movie-card">
```

Comment:
The issues you had with styling are the result of this. A better approach would be to keep that original div with className of movie-card, and then do your link separate.
For example, you can do:
<Link to={`/movies/${movie.id}`} >
<div className='movie-card'>
With that approach, the styling works fine.